helm-charts/basehub/values.schema.yaml

```diff
@@ -282,7 +315,24 @@ properties:
             additionalProperties: false
```

Comment:
Nice, thanks to this we can know that there was not a single typo of gitRepoUrl or similar!